src/index.html

```diff
@@ -1,13 +1,384 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="html">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <title>MyBike</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.ico"
+      type="image/x-icon"
+    >
   </head>
+
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <a href="#home" class="logo">
+          <img
+            src="./images/MyBike-logo-white.svg"
+            class="logo__image"
+            alt="Logo MyBike in white"
+          >
+        </a>
+
+        <div class="navigation__block">
+          <a href="tel:+1 234 555-55-55" class="phone">
```

Comment:
Need to fix the format phone everywhere only in href
```suggestion
          <a href="tel:+12345555555" class="phone">
```